### ISSUE TITLE
Elasticsearch plugin query sample.

### ISSUE THREAD

**Issue by teebu** [alcacoop/elasticsearch-scroll-stream#2]
This took me a little bit to figure out. For elasticsearch module, a query starts with "body": {"query"}

```
      index: 'elasticsearch-test-scroll-stream',
      type: 'test-type',
      search_type: 'scan',
      scroll: '10s',
      size: '50',
      fields: ['name'],
      body: { query: {
        bool: {
          must: [
            {
              query_string: {
                default_field: "_all",
                query: 'name:second*'
              }
            }
          ]
        }
      }
    }
    });
```

**Comment by francov** [comment]
You are right!
As a convention in the `elasticsearch-js` library the `body` identify the data to send along with the request. However this format is not supported by the `elastical` library instead, which I suppose it is in a deprecation status and not maintained anymore.

Furthermore, I will update not only the README but the tests too, bumping and publishing a new version on npm.
Thank you!

**Comment by francov** [comment]
Fixed in 1.0.3
